docs/modules/user-guide/partials/proc_developing-with-devfiles.adoc

```diff
@@ -0,0 +1,36 @@
+[id="proc_developing-with-devfiles_{context}"]
+= Developing with devfiles
+[role="_abstract"]
+
+Develop a node.js “Hello World” application, using the devfile specification. Developing this application introduces you to how a devfile automates and simplifies your development process.
+
+.Prerequisites
+
+* To develop an application by using the devfile specification, you need to be on a cluster. To create a cluster, install link:https://minikube.sigs.k8s.io/docs/start/[minikube start].
```

Comment:
If I'm right, I think we are discouraged to link users to outside pages. Do you think simply saying what they need is sufficient? That is, simply say they need to be on a cluster, need kubectl, need the odo cli tool? 

I thought the links made it easier and quicker for users to get up and going.

Reply:
@jc-berger IIRC, we were discouraged from linking to proprietary and commercial resources while referencing open source projects/docs is considered fine. It could have changed though.

Anyway in your case it seems very helpful to link to a tool that's needed as a prereq.

Reply:
Yes. I think linking to upstream projects is OK. But it's my understanding that linking to Github requires special dispensation. In this case, it's a useful link that will help users get up and running, so I think it's OK.